client/src/components/book/LibrariesStack.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, Button, Paper, Stack, Typography } from '@mui/material';
+import { LibraryViewModelType } from '../../models/LibraryViewModel';
+import { addToCart } from '../../api/cart';
+import useAlert from '../../utils/alerts/useAlert';
+import { useCartStore } from '../../store';
+import AuthorizedView from '../auth/AuthorizedView';
+import { Link } from 'react-router-dom';
+
+function LibrariesStack({ libraries, bookId }: { libraries: LibraryViewModelType[]; bookId: string }) {
+  const { showError, showSuccess } = useAlert();
+  const cartStore = useCartStore();
+  const handleAddToCart = async (bookId: string, libraryId: string) => {
+    try {
+      await addToCart({ bookId, libraryId });
+      showSuccess({ message: 'Dodano do koszyka!' });
+      cartStore.toggleIsChanged();
+    } catch (error) {
```

Comment:
dlaczego nie używamy `useMutation`?

Reply:
Ogólnie skopiowałem to z tego co Kuba wcześniej pisał, mogę próbować zmienic